components/Collections/src/MDCCollectionInfoBarView.h

```diff
@@ -149,14 +156,10 @@ typedef NS_ENUM(NSUInteger, MDCCollectionInfoBarViewStyle) {
 @property(nonatomic, assign) BOOL allowsTap;
 
 /** The optional style that the info bar can be shown. */
-@property(nonatomic, assign) MDCCollectionInfoBarViewStyle style;
+@property(nonatomic, assign) MDCInfoBarStyle style;
```

Comment:
Is this changeable once an info bar has been created or can it be readonly just as *kind* is?